src/scripts.js

```diff
@@ -71,11 +71,14 @@ function onStartup() {
 
 function compilePantryData(recipesList) {
   recipesList.forEach(recipe => {
-    let missingIngredients = pantry.getMissingPartOfRecipe(recipe).ingredients.map(ingredient => {
-      let specificIngredient = ingredientData.find(item => item.id === ingredient.id);
-      return specificIngredient.name;
-    });
-
+    let missingIngredients = pantry.getMissingPartOfRecipe(recipe).ingredients.reduce((acc, ingredient) => {
+      if(ingredient.quantity.amount > 0) {
+        let specificIngredient = ingredientData.find(item => item.id === ingredient.id);
```

Comment:
need to go find name of ingredient (not included in recipe)

Reply:
I noticed that this morning too, could be a new method under recipe class to add name to each ingredient, simple to how we are adding estimated cost!